### PR TITLE
apps/system/cle:  Fix error when compiling with C89 compiler.

### PR DESCRIPTION
**Pull request by patacongo**

### CODE REVIEW

File: system/cle/cle.c [hunk 1]
```diff
@@ -798,6 +799,9 @@ static int cle_editloop(FAR struct cle_s *priv)
 
   for (; ; )
     {
+#if  1 /* Perhaps here should be a config switch */
```
Comment (Ouss4) [inline system/cle/cle.c]: Do you want to keep the PR open, or should it be merged?

**Comment by patacongo** [comment]
I believe it should be merged.  It jsut moves a local variable from the middle of a function to the top as required by C89.  Without this, NSH with the CLE fails to build with the ZDS-II toolchain.

File: system/cle/cle.c (hunk 1 above)
Comment (xiaoxiang781216) [inline system/cle/cle.c]: But @patacongo why we add #if/#endif?

Comment (patacongo) [reply system/cle/cle.c]: It was added.  It was already there.  Just copied.  The original authored intended to add a configuration option but left #if1 in place of the missing option.

**Pull request merged**